src/AssetTransferApi.ts

```diff
@@ -60,11 +62,18 @@ import {
 	LocalTxOpts,
 	Methods,
 	RegistryTypes,
+	ResolvedCallInfo,
 	TransferArgsOpts,
 	TxResult,
 	UnsignedTransaction,
+	XcmBaseArgs,
```

Comment:
I decided to explicitly add the `type` keyword before this type ` { ..., type XcmBaseArgs, ... }` so that its clear its the type and not a value we want for it.